documentation/modules/proc-configuring-kafka-bridge-smr-metrics.adoc

```diff
@@ -0,0 +1,42 @@
+[id='proc-configuring-kafka-bridge-smr-metrics-{context}']
+= Configuring Strimzi Metrics Reporter metrics
+
+[role="_abstract"]
+Enable metrics with Strimzi Metrics Reporter by setting the `bridge.metrics` option to `strimziMetricsReporter`.
+
+.Prerequisites
+
+* xref:proc-downloading-kafka-bridge-{context}[The Kafka Bridge installation archive is downloaded].
+
+.Procedure
+
+. Set the `bridge.metrics` configuration to `strimziMetricsReporter`.
++
+.Configuration for enabling metrics
+
+[source,properties]
+----
+bridge.metrics=strimziMetricsReporter
+----
++
+Optionally, you can configure a comma-separated list of regexes to filter exposed metrics using the `kafka.prometheus.metrics.reporter.allowlist` property.
+If not configured, a default set of metrics will be exposed.
+
+When needed, it is possible to configure the allowlist per client type.
+For example, set `kafka.admin.prometheus.metrics.reporter.allowlist=` to exclude all admin client metrics.
+
+In case the same property is set with multiple prefixes, the most specific prefix wins.
+For example, `kafka.producer.prometheus.metrics.reporter.allowlist` wins over `kafka.prometheus.metrics.reporter.allowlist`. 
+
+You can add any plugin configuration to the Bridge properties file using `kafka.`, `kafka.admin.`, `kafka.producer.`, and `kafka.consumer.` prefixes.
```

Comment:
should we move this above "If required, you can configure an `allowlist` for a specific client type."  as we use `kafka.admin' in the example there?